liquidctl/cli.py

```diff
@@ -45,6 +46,10 @@
   --version                      Display the version number
   --help                         Show this message
 
+Config file options:
+  -f, --file <file>              Select a specific config file to use
```

Comment:
Let's save `-f` for something else, there's no need to spend a one-line option here.

I also don't think `--file` is super clear (anywhere else but this specific place)... what do you think about `--config`?

Reply:
I like that and will update in the next commit